Mathlib/RingTheory/MvPowerSeries/Basic.lean

```diff
@@ -719,6 +720,76 @@ theorem coeff_prod [DecidableEq σ]
       simp only [add_right_inj] at huv
       exact h rfl huv.symm
 
+/-- The `d`th coefficient of a power of a multivariate power series
+is the sum, indexed by `finsuppAntidiag (Finset.range n) d`, of products of coefficients  -/
+theorem coeff_pow [DecidableEq σ] (f : MvPowerSeries σ R) {n : ℕ} (d : σ →₀ ℕ) :
+    coeff R d (f ^ n) =
+      ∑ l in finsuppAntidiag (Finset.range n) d,
+        ∏ i in Finset.range n, coeff R (l i) f := by
+  suffices f ^ n = (Finset.range n).prod fun _ ↦ f by
+    rw [this, coeff_prod]
+  rw [Finset.prod_const, card_range]
+
+/-- The degree of a monomial. -/
+def degree (d : σ →₀ ℕ) : ℕ := d.sum fun _ ↦ id
```

Comment:
This is literally [MvPolynomial.degree](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/MvPolynomial/Homogeneous.html#MvPolynomial.degree) isn't it? Note that even if it is the `MvPolynomial` it is really about `finsupp`. We should really avoid duplicating the API here.

Reply:
I did that already, in another PR!

Reply:
#14875

Reply:
Can you make this PR dependent on the other one?

Reply:
Done.